src/applications/disability-benefits/all-claims/tests/utils/clean-toxic-exposure-data.unit.spec.jsx

```diff
@@ -0,0 +1,1466 @@
+import { expect } from 'chai';
+import cloneDeep from 'platform/utilities/data/cloneDeep';
+import { cleanToxicExposureData } from '../../utils/submit';
+
+describe('cleanToxicExposureData', () => {
+  describe('when user has no toxic exposure data', () => {
+    it('should return the form data unchanged', () => {
+      const formData = {
+        newDisabilities: [{ condition: 'Chronic Bronchitis' }],
+        veteranInformation: { name: 'John Doe' },
+      };
+
+      const result = cleanToxicExposureData(formData);
+
+      expect(result).to.deep.equal(formData);
+    });
+  });
+
+  describe('when user has toxic exposure data but no toxic exposure conditions', () => {
+    it('should return the form data unchanged', () => {
+      const formData = {
+        newDisabilities: [{ condition: 'Chronic Bronchitis' }],
+        toxicExposure: {
+          gulfWar1990: { bahrain: true },
+          herbicide: { vietnam: true },
+        },
+      };
+
+      const result = cleanToxicExposureData(formData);
+
+      expect(result).to.deep.equal(formData);
+    });
+  });
+
+  describe('when user has toxic exposure conditions selected (not "none")', () => {
+    it('should return all toxic exposure data unchanged', () => {
```

Comment:
Not technically, for this and some other scenarios, as there could be other situations influencing the resultant form data (e.g., unselected locations, where the location value is **false**, with corresponding date data), but I understand the intention.